tools/analyzer_plugin/lib/src/async_plugin_apis/diagnostic.dart

```diff
@@ -192,11 +217,39 @@ class _DiagnosticGenerator {
 
     // The analyzer normally filters out errors with "ignore" comments,
     // but it doesn't do it for plugin errors, so we need to do that here.
-    final filteredErrors = filterIgnores(
+    var filteredErrors = filterIgnores(
         collector.errors, unitResult.lineInfo, () => IgnoreInfo.forDart(unitResult.unit!, unitResult.content!));
 
+    filteredErrors = _configureErrorSeverities(filteredErrors);
+
```

Comment:
Is there a reason this can't just be instantiated in a single statement?

```dart
final filteredErrors = _configureErrorSeverities(
  filterIgnores(
    collector.errors, 
    unitResult.lineInfo, 
    () => IgnoreInfo.forDart(unitResult.unit!, unitResult.content!),
  ),
);
```

Reply:
Done